docs/api/contract.rst

```diff
@@ -3,30 +3,50 @@ Contract
 
 .. py:module:: starknet_py.contract
 
-.. autoclass:: Contract
+.. autoclass-with-examples:: Contract
     :members:
     :member-order: groupwise
 
-.. autoclass:: ContractFunction
+----------------
+ContractFunction
+----------------
+
+.. autoclass-with-examples:: ContractFunction
     :exclude-members: __init__, __new__
     :members:
     :member-order: groupwise
 
-.. autoclass:: PreparedFunctionCall
+--------------------
+PreparedFunctionCall
+--------------------
```

Comment:
I would say yes because classes are not visible without headings :/
<img width="793" alt="image" src="https://user-images.githubusercontent.com/61014013/216257577-fc9c4022-d003-4eaa-86c2-1c13c5c45f0f.png">